test/framework/functional/vector/deploy.go

```diff
@@ -0,0 +1,50 @@
+package vector
```

Comment:
Is this cruft some a previous PR?

Reply:
yes, i was working on vector functional test earlier. the file got added by mistake

Reply:
Please remove before merge

Reply:
removed the file

Reply:
?? i pushed a commit with file deleted

Reply:
i dont understand, i deleted this file, and this file does not exist when i have staged the files.
But when i create the commit, this file gets created
```
vimalkum (vector-es-add-id +) cluster-logging-operator $ git commit -m "Add "_id" to ES vector config; enable ES vector tests"
Dockerfile  << cannot modify Dockerfile
Dockerfile.in
bundle/manifests/cluster-logging.v5.4.0.clusterserviceversion.yaml
internal/generator/vector/conf_test.go
internal/generator/vector/output/elasticsearch/elasticsearch.go
internal/generator/vector/output/elasticsearch/elasticsearch_test.go
internal/generator/vector/output/elasticsearch/output_conf_es_test.go
internal/generator/vector/pipelines_to_output.go
manifests/5.4/cluster-logging.v5.4.0.clusterserviceversion.yaml
test/e2e/logforwarding/elasticsearchmanaged/clo_managed_logstore_test.go
test/e2e/logforwarding/elasticsearchmanaged/forwarding_with_defaults_test.go
test/framework/e2e/framework.go
test/helpers/clusterlogging.go
[vector-es-add-id 9a284379] Add _id to ES vector config; enable ES vector tests
 15 files changed, 278 insertions(+), 353 deletions(-)
 create mode 100644 test/framework/functional/vector/deploy.go
 delete mode 100644 test/helpers.go
```

Reply:
the files does not exist in local also
```
$ git rm test/framework/functional/vector/deploy.go
fatal: pathspec 'test/framework/functional/vector/deploy.go' did not match any files
$ rm test/framework/functional/vector/deploy.go
rm: cannot remove 'test/framework/functional/vector/deploy.go': No such file or directory
```

Reply:
may be my git index is corrupted?

Reply:
managed to fix it by moving changed to new branch and renaming new branch back to this one.

Please lgtm